src/cf-pagination/src/molecules/pagination.less

```diff
@@ -8,8 +8,8 @@
     &_form {
         padding: unit( 5px / @base-font-size-px, em );
         border-radius: unit( 4px / @base-font-size-px, em );
-        background: @pagination-bg-color;
-        color: @pagination-text-color;
```

Comment:
I vaguely recall a discussion around including `-color` since `@pagination-text` on its own doesn't convey what it's doing (e.g. it could just as easily be a placeholder for the font size). But whatever is the precedent around here should be what we go with.

Reply:
yep i agree but i checked other components and we don't specify with such suffixes generally. a change i would be into, though.